29th_first_semina/app/src/main/java/com/example/a29th_first_semina/SignInActivity.kt

```diff
@@ -0,0 +1,62 @@
+package com.example.a29th_first_semina
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.Toast
+import com.example.a29th_first_semina.databinding.ActivitySigninBinding
+
+class SignInActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySigninBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySigninBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        initButtonClickEvent()
+
+    }
+
+    private fun initButtonClickEvent() {
+
+        clickLoginEvent()
+        clickSignUpEvent()
+    }
+
+    private fun moveHomeActivity() {
+        val intent = Intent(this, HomeActivity::class.java)
+        startActivity(intent)
+    }
+
+    private fun moveSignUpActivity() {
+        val signupintent = Intent(this, SignUpActivity::class.java)
```

Comment:
두 단어로 구성된 변수는 camelCase로 적어주는게 좋습니다.
```suggestion
        val signupIntent = Intent(this, SignUpActivity::class.java)
```

Reply:
엇 여기 왜 camelCase로 안되어있지 감쟈합니다